sygnal/webpushpushkin.py

```diff
@@ -75,7 +85,7 @@ class WebpushPushkin(ConcurrencyLimitedPushkin):
         "ttl",
     } | ConcurrencyLimitedPushkin.UNDERSTOOD_CONFIG_FIELDS
 
-    def __init__(self, name, sygnal, config):
+    def __init__(self, name: str, sygnal: Sygnal, config: Dict[str, Any]):
```

Comment:
`Sygnal` may need to be quoted since it's conditionally imported.
```suggestion
    def __init__(self, name: str, sygnal: "Sygnal", config: Dict[str, Any]):
```